lib/workers/branch/parent.js

```diff
@@ -39,7 +39,7 @@ async function getParentBranch(config) {
     if (pr.canRebase) {
       logger.info(`Branch is not mergeable and needs rebasing`);
       // TODO: Move this down to api library
-      if (config.isGitLab) {
+      if (config.isGitLab || config.isVsts) {
```

Comment:
If you delete the branch of an existing PR, does VSTS still kees the corresponding PR open?

Reply:
Yes, the PR is still open with a message: `The source branch has been deleted.`.
When you recreate the branch, it will be automatically link to the PR.
If you want you can abandon the PR, anb both branch and PR will be recreated.

So I think it's good like this